pkg/operator/starter.go

```diff
@@ -531,9 +531,6 @@ func prepareOauthAPIServerOperator(ctx context.Context, controllerContext *contr
 		[]string{
 			"oauth-apiserver/ns.yaml",
 			"oauth-apiserver/apiserver-clusterrolebinding.yaml",
-			"oauth-apiserver/authenticator-sa.yaml",
-			"oauth-apiserver/authenticator-clusterrole.yaml",
-			"oauth-apiserver/authenticator-clusterrolebinding.yaml",
```

Comment:
are they cleaned up anywhere?

Reply:
they were never there in 4.7 so there's no need for a cleanup